### DISCUSSION TITLE
Version 1.17.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#179] [Announcements]
- tags are a new way to organize your apps
  - you can assign as many tags as you want to apps (and other search results)
  - tags are used as keywords in the search, making it easier to find items that have tags assigned
  - tags can also be pinned to the favorites section for even faster access
- favorites have been improved
  - it's now configurable if and how many frequently used items are shown in the favorites section
  - items can be grouped using tags
  - the 'edit favorites' sheet has been redesigned
  - it's now possible to create shortcuts from the launcher (previously shortcuts could only be created from within apps and some app shortcuts were unavailable because of this)
- the 'edit widgets' button can now be hidden
- probably bugfixes and other stuff, I forgor 💀

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.17.0'>Version 1.17.0</a>.</em>

**Comment by Tealk** [comment]
The tags are great, it would have been nice if you could select a tag and then add a list of apps. That would be a little faster

**Reply by MM2-0** [reply]
I think currently the fastest way to assign many apps to a tag is to add both the tag and all the apps to favorites and then drag and drop the apps to the tag in the edit favorites sheet.

But yes, I agree, there is still room for improvement (things like renaming tags would also be nice I think) and I'll probably expand on that in further updates in the future.

**Reply by Tealk** [reply]
Cool did not know that you can drag and drop to sort tags
